docs/introduction/open-source/public_code.md

```diff
@@ -92,108 +96,127 @@ Reviews MUST include source, policy, tests and documentation. | yes | repo is co
-Documentation SHOULD aim for a lower secondary education reading level, as recommended by the [Web Content Accessibility Guidelines 2](https://www.w3.org/WAI/WCAG21/quickref/?showtechniques=315#readable). | | Would be good to get an evaluation of this prior to investing in translation.
-Any code, documentation or tests MAY have a translation. |
+Documentation SHOULD aim for a lower secondary education reading level, as recommended by the [Web Content Accessibility Guidelines 2](https://www.w3.org/WAI/WCAG21/quickref/?showtechniques=315#readable). |  | Would be good to get an evaluation of this prior to investing in translation.
+Providing a translation of any code, documentation or tests is OPTIONAL. |  |
 
 ## [Use open standards](https://standard.publiccode.net/criteria/open-standards.html)
 
-- [x] compliant with this criterion.
+- [x] criterion met.
 
 Requirement | meets | links and notes
 -----|-----|-----
-For features of a codebase that facilitate the exchange of data the codebase MUST use an open standard that meets the Open Source Initiative Open Standard Requirements. | yes | The [Zaakgericht Werken in het Gemeentelijk Gegevenslandschap](https://www.gemmaonline.nl/images/gemmaonline/f/f6/20190620_-_Zaakgericht_werken_in_het_Gemeentelijk_Gegevenslandschap_v101.pdf) meets the 5 criteria of the [Open Standards Requirement for Software](https://opensource.org/osr)
+For features of a codebase that facilitate the exchange of data the codebase MUST use an open standard that meets the [Open Source Initiative Open Standard Requirements](https://opensource.org/osr). | yes | The [Zaakgericht Werken in het Gemeentelijk Gegevenslandschap](https://www.gemmaonline.nl/images/gemmaonline/f/f6/20190620_-_Zaakgericht_werken_in_het_Gemeentelijk_Gegevenslandschap_v101.pdf) meets the 5 criteria of the [Open Standards Requirement for Software](https://opensource.org/osr)
+Any non-open standards used MUST be recorded clearly as such in the documentation. | n/a |
+Any standard chosen for use within the codebase MUST be listed in the documentation with a link to where it is available. | yes | <!-- link? -->
+Any non-open standards chosen for use within the codebase MUST NOT hinder collaboration and reuse. | n/a |
 If no existing open standard is available, effort SHOULD be put into developing one. | n/a |
-Standards that are machine testable SHOULD be preferred over those that are not. | yes | see test suite
-Functionality using features from a non-open standard (one that doesn’t meet the Open Source  Initiative Open Standard Requirements) MAY be provided if necessary, but only in addition to compliant features. |
-All non-compliant standards used MUST be recorded clearly in the documentation. | n/a |
-The codebase SHOULD contain a list of all the standards used with links to where they are available. | yes |
+Standards that are machine testable SHOULD be preferred over those that are not. |  | <!-- need to check -->
```

Comment:
We have OAS tests and Integration tests for the API schema's/interaction in the CI pipeline, using Postman/newman for automation.